### DISCUSSION TITLE
If I want train my own model?

### DISCUSSION THREAD

**Discussion by marcussacana** [DanRuta/xVA-Synth#10] [Q&A] [answered]
1 - How can I train the my own model?
2 - This will works only for english text?

**Comment by DanRuta** [comment, marked as answer]
This works for any language that models are trained on. To train your own models, check out NVIDIA's FastPitch github repo

**Reply by BaaridunNasr** [reply]
Would this mean that we can use our own audio samples to train and synthesize speech?

I wanted to look into working on FromSoft games starting with Sekiro.

**Reply by DanRuta** [reply]
Yes, that should be fine. You just need to compile your speech dataset first. Make sure the data is as clean as possible, without any other audio, like music, sfx, and such.

For the best sentence quality, you should also train a Tacotron2 model first, and then use that model for pre-processing the data for FastPitch. And finally, train your own vocoder model for audio quality.

You can join the Discord server (technical-chat channel) for discussions around this
